server/src/main/java/org/apache/druid/server/QueryResource.java

```diff
@@ -83,6 +83,7 @@
 
   protected static final int RESPONSE_CTX_HEADER_LEN_LIMIT = 7 * 1024;
 
+  public static final String HEADER_RESPONSE_CONTEXT = "X-Druid-Response-Context";
   public static final String HEADER_IF_NONE_MATCH = "If-None-Match";
   public static final String HEADER_ETAG = "ETag";
```

Comment:
Duplicates a field in `ResponseContext`

Reply:
Renamed `CTX_HEADER_ETAG` to `CTX_ETAG`. It still has the same value as `QueryResource.HEADER_ETAG` but serves a different goal (header name vs context key)